block-storage-attacher/tests/e2e/scripts/common.sh

```diff
@@ -295,10 +317,10 @@ function rm_cluster {
     removed=1
     cluster_name=$1
 
-    bx cs clusters
+    ibmcloud ks clusters
 
     for i in {1..3}; do
-        if bx cs cluster-rm $cluster_name -f; then
+        if ibmcloud ks cluster-rm $cluster_name -f; then
```

Comment:
This command is `ibmcloud ks cluster rm`

Reply:
This also we are not suing changed this one